rest/server_context.go

```diff
@@ -538,8 +538,18 @@ func (sc *ServerContext) _getOrAddDatabaseFromConfig(config *DbConfig, useExisti
 
 	if config.RevsLimit != nil && *config.RevsLimit > 0 {
 		dbcontext.RevsLimit = *config.RevsLimit
+		if dbcontext.RevsLimit < 20 {
+			return nil, fmt.Errorf("The revs_limit (%v) configuration cannot be set lower than 20.", dbcontext.RevsLimit)
+		}
+
+		if dbcontext.RevsLimit < 100 {
```

Comment:
I feel like 100 is too high for a 'not recommended' warning - if users know their update frequency is low, it's perfectly reasonable to take revs_limit down to 20.  

Maybe change the phrasing to 'Setting the revs_limit to less than 100 may have unwanted results when documents are frequently updated.  Please see documentation for details.'